src/NHibernate.Test/NHSpecificTest/NH2195/SQLiteMultiCriteriaTest.cs

```diff
@@ -144,5 +144,63 @@ public void MultiCriteriaQueriesWithStringsShouldExecuteCorrectly()
 				Assert.AreEqual(1, list.Count);
 			}
 		}
+
+		[Test]
+		public void QueryBatchWithIntsShouldExecuteCorrectly()
+		{
+			var driver = Sfi.ConnectionProvider.Driver;
+			if (!driver.SupportsMultipleQueries)
+				Assert.Ignore("Driver {0} does not support multi-queries", driver.GetType().FullName);
```

Comment:
Are you sure we want to skip such tests (that are not checking that single query is executed for batch but some data conditions retrieval)? As query batch doesn't throw maybe it's good to test that it also works adequeatly when `driver.SupportsMultipleQueries == false`.

Reply:
I have port existing multi-query/criteria tests without changing their conditions. Many of them would fail with drivers not supporting multiple queries, because they would see more queries executed than expected.